### PR TITLE
Implement get supported mds languages set mds language

### PR DESCRIPTION
**Pull request by antipovi-draeger**
Implemet manipulations to get and set supported languages

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: src/t2iapi/device/device_responses.proto [hunk 1]
```diff
@@ -15,6 +15,14 @@ import "t2iapi/biceps/metadata.proto";
 option java_package = "com.draeger.medical.t2iapi.device";
 option java_outer_classname = "DeviceResponses";
 
+/*
+Response containing all languages supported by the UI component represented by a MDS
+ */
+message GetMdsUiSupportedLanguagesResponse {
+    BasicResponse status = 1;
+    repeated string languages = 2;  // list of languages which are supported by the UI component represented by a MDS
```
Comment (ldeichmann) [inline src/t2iapi/device/device_responses.proto:23]: The `SetMdsUiLanguageRequest` restricts languages to RFC5646, does this also apply to these languages?

File: src/t2iapi/device/device_responses.proto [hunk 2]
```diff
@@ -15,6 +15,14 @@ import "t2iapi/biceps/metadata.proto";
 option java_package = "com.draeger.medical.t2iapi.device";
 option java_outer_classname = "DeviceResponses";
 
+/*
+Response containing all languages supported by the UI component represented by a MDS
```
Comment (ldeichmann) [inline src/t2iapi/device/device_responses.proto:19]:
```suggestion
Response containing all languages supported by the UI component represented by an MDS.
```

File: src/t2iapi/device/device_responses.proto (hunk 1 above)
Comment (antipovi-draeger) [reply src/t2iapi/device/device_responses.proto:23]: they should, since those can be directly used for SetMdsUiLanguageRequest manipulation

Comment (ldeichmann) [reply src/t2iapi/device/device_responses.proto:23]: Then it should also be mentioned in this comment

Comment (antipovi-draeger) [reply src/t2iapi/device/device_responses.proto]: done

File: src/t2iapi/device/device_responses.proto (hunk 2 above)
Comment (antipovi-draeger) [reply src/t2iapi/device/device_responses.proto]: done

Comment (ldeichmann) [reply src/t2iapi/device/device_responses.proto]: Not really, you applied half of my change manually, it's missing the other half.

**Review by ldeichmann — APPROVED** [review]

**Review by belagertem — APPROVED** [review]

**Pull request merged**